### DISCUSSION TITLE
installazione dongle zigbee e ZHA

### DISCUSSION THREAD

**Discussion by Starc73** [indomus/forum#3199] [HUB: Home Assistant]
Salve a tutti,
Ho un NUC con HASSOS e ho comprato una chiavetta Sonoff ZigBee 3.0 Dongle Plus v2, e ho seguito la guida ZHA e quindi l'ho flashato con il fw zigbee presente nella procedura guidata da web app; su home assistant il sw viene visto,  ho modificato il file di configurazione con un canale, quando vado tra i dispositivi a chiave viene riconosciuta ma  premendo configura mi fornisce il un errore "unknown error occurred"

![image](https://github.com/user-attachments/assets/7f528078-4b9a-4263-8be9-5e8d959ab906)

Allego il log di home assistant
[homeassistant.log](https://github.com/user-attachments/files/16778420/homeassistant.log)

Avete qualche consiglio in merito ?
Grazie

**Comment by snakuzzo** [comment]
Che firmware hai messo ?
Che vuol dire "ho modificato il file di configurazione con un canale" ?

In ogni caso l'errore è questo...

```
EzspStatus.ERROR_INVALID_ID
```

Se non ne esci devi andare [QUI](https://github.com/zigpy/bellows/issues) e cercare
